gluten-core/src/main/scala/io/glutenproject/expression/BinaryExpressionTransformer.scala

```diff
@@ -51,12 +50,40 @@ class BinaryExpressionTransformer(
   }
 }
 
+class LikeTransformer(substraitExprName: String,
+                      left: ExpressionTransformer,
+                      right: ExpressionTransformer,
+                      original: Expression)
+  extends ExpressionTransformer with Logging {
+  override def doTransform(args: java.lang.Object): ExpressionNode = {
+    val leftNode = left.doTransform(args)
+    val rightNode = right.doTransform(args)
+    val escapeCharNode = ExpressionBuilder.makeLiteral(
+      original.asInstanceOf[Like].escapeChar.toString, StringType, false)
+
+    val functionMap = args.asInstanceOf[java.util.HashMap[String, java.lang.Long]]
+    val functionId = ExpressionBuilder.newScalarFunction(
+      functionMap,
+      ConverterUtils.makeFuncName(
+        substraitExprName,
+        original.children.map(_.dataType),
+        FunctionConfig.OPT))
+
+    val expressionNodes = Lists.newArrayList(leftNode, rightNode, escapeCharNode)
```

Comment:
CH backend can not support escapeChar parameter, can we add this parameter according to the backend first?

Reply:
Got it. Removed escapeChar for CH backend.